src/syngen/ml/metrics/metrics_classes/metrics.py

```diff
@@ -1079,20 +1085,23 @@ def calculate_all(self, categ_columns: List[str], cont_columns: List[str]):
         if best_binary is not None:
             logger.info(
                 f"The ratio of synthetic binary accuracy to original is "
-                f"{round(synth_score_binary/score_binary, 3)}. The model considers "
-                f"the {best_binary} column as a target and other columns as predictors"
+                f"{round(synth_score_binary/score_binary, 4)}. "
+                f"The model considers the {best_binary} column "
+                f"as a target and other columns as predictors"
             )
         if best_categ is not None:
             logger.info(
                 f"The ratio of synthetic multiclass accuracy to original is "
-                f"{round(synth_score_categ/score_categ, 3)}. The model considers "
-                f"the {best_categ} column as a target and other columns as predictors"
+                f"{round(synth_score_categ/score_categ, 4)}. "
+                f"The model considers the {best_categ} column "
```

Comment:
To avoid some duplication you can put the repeated part of log messages into the variable like this - 

common_message = "The model considers the {} column as a target and other columns as predictors"

log_message_1 = common_message.format(best_binary)
log_message_2 = common_message.format(best_categ)